tests/test_simulate.py

```diff
@@ -15,6 +15,8 @@
 #   limitations under the License.
 
 import os
+import cgpm
+import crosscat
```

Comment:
Do you need these two imports? also, they are not standard library modules, so should go in the block from `crosscat.LocalEngine`.